charts/linkerd2-cni/templates/cni-plugin.yaml

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 This file was inspired by
 1) https://github.com/istio/cni/blob/c63a509539b5ed165a6617548c31b686f13c2133/deployments/kubernetes/install/manifests/istio-cni.yaml
 */ -}}
-
+{{- if .Values.createNamespace -}}
```

Comment:
It seems to me that before the change the namespace was automatically created. What do you think about changing the logic here, something like:

```
{{- if not .Values.skipNamespace}}
 kind: Namespace
 ...
```
 
In addition, this would mean changing the flag name from 'createNamespace' to something like `skip-namespace-creation` or `omit-namespace`. WDYT @alex-berger @Pothulapati